Core/src/main/java/com/theprogrammingturkey/comz/listeners/WeaponListener.java

```diff
@@ -199,7 +199,7 @@ public void onGrenade(PlayerInteractEvent event)
 				item.setVelocity(player.getLocation().getDirection().multiply(1));
 				item.setPickupDelay(1000);
 
-				COMZombies.scheduleTask(140, () ->
+				COMZombies.scheduleTask(100, () ->
```

Comment:
What's the reason for lowering this?

Reply:
Monkey bombs explodes after 7 seconds, grenades after 5 seconds, you both use 7 seconds for it, and thats not correct